client/state/selectors/get-gutenberg-editor-url.js

```diff
@@ -8,6 +8,7 @@ import { getEditorPath } from 'state/ui/editor/selectors';
 import { isEnabled } from 'config';
 import isSiteAtomic from 'state/selectors/is-site-automated-transfer';
 import { addQueryArgs } from 'lib/route';
+import hasWpAdminEditorConflictingPlugin from 'state/selectors/has-wp-admin-editor-conflicting-plugin';
```

Comment:
I don't like the selector being so specific (at least in name); it should be usable anywhere in Calypso, with any array of slugs for whatever purpose, I think.